composer.json

```diff
@@ -31,6 +31,11 @@
         "symplify/easy-coding-standard": "^10.0.6",
         "nikic/php-parser": "^4.0"
     },
+    "autoload": {
+        "psr-4": {
+            "PhpCodingStandards\\": "src"
```

Comment:
Maybe this should be `Jump24\\PhpCodingStandards\\` just to be really clear when importing bits that it's from this package and not another?

Reply:
I made it JumpTwentyFour to match the laravel coding standards repo :)